app/plugins/Awards/templates/Recommendations/view.php

```
@@ -19,22 +19,9 @@
-        [
-            "confirm" => __(
-                "Are you sure you want to delete recommendation for {0}?",
-                h($recommendation->member_sca_name . ' about ' . $recommendation->award->name),
-            ),
-            "title" => __("Delete"),
-            "class" => "btn btn-danger btn-sm",
-        ],
-    ); ?>
+<button type="button" class="btn btn-primary btn-sm edit-rec" data-bs-toggle="modal" data-bs-target="#editModal"
+    data-controller="outlet-btn" data-action="click->outlet-btn#fireNotice"
+    data-outlet-btn-btn-data-value='{ "id":<?= $recommendation->id ?>}'>Edit</button>
```

PHPStan (2.1.46)

[error] Access to an undefined property Awards\Model\Entity\Recommendation::$id.
Learn more: https://phpstan.org/blog/solving-phpstan-access-to-undefined-property

(property.notFound)